src/main/java/configuration/ThreadCountConfig.java

```diff
@@ -0,0 +1,24 @@
+package configuration;
+
+import java.util.regex.Pattern;
+
+/***
+ * Set up the number of test run threads
+ */
+public class ThreadCountConfig {
+
+    private static final Pattern isInteger = Pattern.compile("^\\d+$");
```

Comment:
Update pattern name. Boolean variables start with 'is' and constants should be upper-case

Reply:
fixed